provisioning/playbooks/couchbase/create_bucket.yml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: couchbase
+  sudo: False
+  gather_facts: False
+  tasks:
+    - name: Create cluster grouping
+      group_by: key={{ couchbase_server_node_role }}
+
+- hosts: primary
+  sudo: False
+  gather_facts: False
+  vars:
```

Comment:
Yes, `bucket_ram <= server_ram`, but `bucket_name` is not related to `default_bucket`. `default_bucket` wasn't used, so it is being removed. This file creates a single bucket with the variables mentioned specifically for this bucket.